Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormLawRegistered.cshtml.cs

```diff
@@ -13,7 +14,7 @@ public class LegalFormLawRegisteredModel(
     IOrganisationClient organisationClient) : PageModel
 {
     [BindProperty]
-    [Required(ErrorMessage = "Please enter the law under which your organisation is registered")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Supplier_LegalFormLawRegistered_ErrorMessage), ErrorMessageResourceType = typeof(StaticTextResource))]
```

Comment:
`ErrorMessageResourceName` not `ErrorMessage`

Reply:
Done